modules/criteoBidAdapter.js

```diff
@@ -25,7 +25,7 @@ const LOG_PREFIX = 'Criteo: ';
   Unminified source code can be found in the privately shared repo: https://github.com/Prebid-org/prebid-js-external-js-criteo/blob/master/dist/prod.js
 */
 const FAST_BID_VERSION_PLACEHOLDER = '%FAST_BID_VERSION%';
-export const FAST_BID_VERSION_CURRENT = 135;
+export const FAST_BID_VERSION_CURRENT = 136;
```

Comment:
please commit to https://github.com/Prebid-org/prebid-js-external-js-criteo/tree/master/dist/ first

Reply:
Hello, thanks for the review. https://github.com/Prebid-org/prebid-js-external-js-criteo/tree/master/dist/136 the commit to update fast bid script has been done.